cellbase-core/src/main/java/org/opencb/cellbase/core/ParamConstants.java

```diff
@@ -113,8 +113,8 @@ public Type type() {
     public static final String VERSION_DESCRIPTION = "API version, e.g.: " + DEFAULT_VERSION;
 
     public static final String DATA_RELEASE_PARAM = "dataRelease";
-    public static final String DATA_RELEASE_DESCRIPTION = "Data release. To use the default data release, set this to 0. To get the list"
-            + " of available data release, please call the endpoint 'meta/dataReleases'";
+    public static final String DATA_RELEASE_DESCRIPTION = "Data release. To get the list of available data release, please call the"
```

Comment:
You are missing one 's' in the description. ...To get the list of available data release**s**, please call...